lib/Echidna/Output/Source.hs

```diff
@@ -39,59 +40,26 @@ saveCoverage Nothing  _  _  _ = pure ()
 ppCoveredCode :: SourceCache -> [SolcContract] -> CoverageMap -> Text
 ppCoveredCode sc cs s | s == mempty = "Coverage map is empty"
                       | otherwise   =
-  let allLines = M.toList $ sc ^. sourceLines
+  let allFiles = zipWith (\(srcPath, _) srcLines -> (srcPath, V.map decodeUtf8 srcLines))
```

Comment:
we can prefilter with `fst` so we can avoid the messy lambda. e.g. `over _2 (V.map decodeUtf8) $ zip (fst <$> sc ^. sourceFiles) (sc ^. sourceLines)`.

Reply:
maybe even `sc ^.. sourceFiles . traverse . _1`

Reply:
I decided to leave it as is, the lambda is just one line and I feel it's pretty clear what's going on here.